iocore/net/Makefile.am

```diff
@@ -73,13 +76,11 @@ libinknet_a_SOURCES = \
   P_Socks.h \
   P_SSLCertLookup.h \
   P_SSLConfig.h \
-  P_SSLNetAccept.h \
   P_SSLNetProcessor.h \
-  P_SSLNetVConnection.h \
-  P_SSLNextProtocolAccept.h \
-  P_SSLNextProtocolSet.h \
+  I_SSLNextProtocolSet.h \
```

Comment:
Here, I'm rename P_SSLNextProtocolSet.h to I_SSLNextProtocolSet.h.
It expose the "SSLNextProtocolSet" for "ProtocolProbeSessionAccept" in proxy directory.
The dup one removed.